diff --git a/constraints.txt b/constraints.txt
index d6c2267..0195039 100644
--- a/constraints.txt
+++ b/constraints.txt
@@ -1,7 +1,2 @@
-babel # Required by distributedcloud-client
-openstacksdk===0.15.0
-cloudify-common==6.4.2
-PrettyTable<0.8,>=0.7.2 # Required by distributedcloud-client
-python-keystoneclient==3.21.0
 git+https://github.com/starlingx/config.git@4.0.0#egg=cgtsclient&subdirectory=sysinv/cgts-client/cgts-client
 git+https://github.com/starlingx/distcloud-client.git@4.0.0#egg=distributedcloud-client&subdirectory=distributedcloud-client
diff --git a/setup.py b/setup.py
index fffab87..ee864b2 100644
--- a/setup.py
+++ b/setup.py
@@ -33,13 +33,15 @@ setup(
     author="Cloudify.Co",
     author_email="cosmo-admin@cloudify.co",
     packages=[
+        'pydoc',
+        'webbrowser',
         'cloudify_starlingx',
         'cloudify_starlingx_sdk',
     ],
     license="LICENSE",
     description="Represent StarlingX Workloads in Cloudify.",
     install_requires=[
-        'cloudify-common>=6.4.2,<7.0.0',
+        'cloudify-common>=7.0.0',
         'distributedcloud-client',
         'cgtsclient',
         'httplib2',
